test/jdk/sun/security/x509/AlgorithmId/NonStandardNames.java

```diff
@@ -62,6 +62,10 @@ public static void main(String[] args) throws Exception {
             new PKCS9Attribute(PKCS9Attribute.MESSAGE_DIGEST_OID, md.digest(data)),
         });
 
+        // test PKCS9Attributes.toString(), PKCS9Attributes.getAttributes()
+        System.out.println(authed);
+        authed.getAttributes();
```

Comment:
Looks like the old `getAttributes()` would only throw NPE if one of the attribute is of a type after `PKCS9_OIDS[10]`.